gcpy/species_database.yml

```diff
@@ -136,6 +136,55 @@ ALK4:
   Is_Advected: true
   Is_Gas: true
   MW_g: 58.12
+aoa:
+  FullName: Age of air uniform source tracer
+  Is_Advected: true
+  Is_Gas: true
+  Is_Tracer: true
+  Snk_Horiz: all
+  Snk_Mode: constant
+  Snk_Value: 0
+  Snk_Vert: surface
+  Src_Add: true
+  Src_Horiz: all
+  Src_Mode: constant
+  Src_Units: timestep
```

Comment:
For units, see comment by @lizziel at https://github.com/geoschem/geos-chem/pull/1816#discussion_r1212282154

Reply:
Also see link to @lizziel's comment at https://github.com/geoschem/geos-chem/pull/1816#discussion_r1212246849

Reply:
The species_database.yml file has now been updated in both the geos-chem repository and here